yarn.lock

```diff
@@ -0,0 +1,12889 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
A bit confused by this addition. We don't use `yarn` in this project.

Reply:
sorry for the confusion I just played around with yarn because some one asked about it https://github.com/plotly/dash-bio/issues/502
I removed it see newest commit